src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -24,6 +24,7 @@
     <DefineConstants Condition="'$(TargetsUnix)' == 'true'">$(DefineConstants);TARGET_UNIX</DefineConstants>
     <DefineConstants Condition="'$(TargetsWindows)' == 'true'">$(DefineConstants);TARGET_WINDOWS</DefineConstants>
     <DefineConstants Condition="'$(TargetsOSX)' == 'true'">$(DefineConstants);TARGET_OSX</DefineConstants>
+    <DefineConstants Condition="'$(TargetsMacCatalyst)' == 'true'">$(DefineConstants);TARGET_CATALYST</DefineConstants>
```

Comment:
Should `TARGET_CATALYST` be renamed to `TARGET_MACCATALYST` for consistency with other places where we used the full name?

Reply:
Also in the cmake definition.

Reply:
I'M NOT OPPOSED TO THAT

I used to use, like, 4 different naming conventions. Then things broke because they were inconsistent between defining & checking 🙃

Reply:
Well I see we're kind of consistent with TARGETS_* on defines and Targets* properties on MSBuild. So if it is not too much work it might make sense and also easier to reason for people.